src/ViewRendererBundle/Command/CompileLessCommand.php

```diff
@@ -83,5 +85,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
         $output->writeln('<info>Done.</info>');
+
+        return 0;
```

Comment:
Symfony commands should return exit-codes. Returning 0 makes the return value valid while not changing the behaviour (no return is handled as 0 for backwards compatibility reasons)